server/src/main/java/io/pravega/schemaregistry/storage/StoreExceptions.java

```diff
@@ -73,9 +73,10 @@ public static StoreExceptions create(final Type type, final String errorMessage)
      * @param cause        Exception cause.
      * @param errorMessage The detailed error message.
      * @return Instance of type of StoreException.
+     * @throws IllegalArgumentException
      */
     public static StoreExceptions create(final Type type, final Throwable cause, final String errorMessage) {
-        Preconditions.checkArgument(cause != null || (errorMessage != null && !errorMessage.isEmpty()),
+        Preconditions.checkArgument(cause != null || errorMessage != null && !errorMessage.isEmpty(),
```

Comment:
why is this change required?

Reply:
As we are changing the Checkstyle version it was claimed by that, it was claiming that `redundant Pair of parenthesis`